ansible/templates/etc/systemd/system/koala-check-study-year.service.j2

```diff
@@ -5,7 +5,7 @@ Description=Check for creation of new study year for Constipated Koala
 OnFailure=failure-notificator@%n.service
 
 [Service]
-Type=simple
+Type=oneshot
```

Comment:
`Type=oneshot: this is useful for scripts that do a single job and then exit. You may want to set RemainAfterExit=yes as well so that systemd still considers the service as active after the process has exited.`  from https://wiki.archlinux.org/index.php/systemd